src/server/sendLayoutHTTPResponse.js

```diff
@@ -266,7 +267,13 @@ function serializeApplication({
     assetStream = renderError(appName, err);
   }
   assetsStream.add(assetStream);
-  bodyStream.add(stringStream(`<div id="single-spa-application:${appName}">`));
+  bodyStream.add(
+    stringStream(
+      appClassNames
+        ? `<div id="single-spa-application:${appName}" class="${appClassNames}">`
+        : `<div id="single-spa-application:${appName}">`
```

Comment:
I'd prefer not to duplicate the beginning part of the string in both conditional branches here:

```suggestion
      `<div id="single-spa-applicaiton:${appName}" ${appClassName ? `class="${appClassName}"` : ""}></div>`
```

Reply:
Totally agree, will change. Actually reverted from your suggestion to what is in the PR for readability, so now I'll change back. 😅